src/DDTrace/Contracts/Tracer.php

```diff
@@ -127,4 +127,9 @@ public function flush();
      * @return mixed
      */
     public function getPrioritySampling();
+
+    /**
+     * @return Scope|null
+     */
+    public function getRootSpan();
```

Comment:
Is  there a  reason why  we don't want to add `startRootSpan()` method to the contract?  :)